README.md

```diff
@@ -43,56 +15,37 @@ operations:
 The convolution operations have several implementations, including tiled and
 Winograd kernels. The supported data format is NHWC.
 
-## Citation
+The project is maintained by [Codeplay Software](https://developer.codeplay.com).
 
-If you use SYCL-DNN in your research, please cite the library as follows:
+## Supported Platforms
 
-> Rod Burns, John Lawson, Duncan McBain, and Daniel Soutar. 2019. *Accelerated
-> Neural Networks on OpenCL Devices Using SYCL-DNN.* In Proceedings of the
-> International Workshop on OpenCL (IWOCL'19). ACM, New York, NY, USA, Article
-> 10, 4 pages. DOI: https://doi.org/10.1145/3318170.3318183
-
-```bibtex
-@inproceedings{Burns:2019:ANN:3318170.3318183,
- author = {Burns, Rod and Lawson, John and McBain, Duncan and Soutar, Daniel},
- title = {Accelerated Neural Networks on OpenCL Devices Using SYCL-DNN},
- booktitle = {Proceedings of the International Workshop on OpenCL},
- series = {IWOCL'19},
- year = {2019},
- isbn = {978-1-4503-6230-6},
- location = {Boston, MA, USA},
- pages = {10:1--10:4},
- articleno = {10},
- numpages = {4},
- url = {http://doi.acm.org/10.1145/3318170.3318183},
- doi = {10.1145/3318170.3318183},
- acmid = {3318183},
- publisher = {ACM},
- address = {New York, NY, USA},
- keywords = {GPGPU, OpenCL, SYCL, machine learning, neural networks},
-}
-```
-
-## Requirements
+The master branch of SYCL-DNN is regularly tested with the "Supported" hardware 
+listed on [the ComputeCpp Supported Platforms page](https://developer.codeplay.com/products/computecpp/ce/guides/platform-support).
 
+SYCL-DNN may also work on other hardware and platforms assuming they implement 
```

Comment:
Is there any reason why this is a new paragraph? It should probably be reflowed into the previous. Actually that's true for all sentences in this section: Why can they not all be merged into one paragraph?